components/library/libass/Makefile

```diff
@@ -18,23 +18,28 @@ BUILD_BITS=			32_and_64
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		libass
-COMPONENT_VERSION=	0.15.2
+COMPONENT_VERSION=	0.16.0
+COMPONENT_REVISION=	1
 COMPONENT_FMRI=		library/video/libass
```

Comment:
COMPONENT_REVISION is superfluous here. It only need to be set/incremented if COMPONENT_VERSION doesn't change. If COMPONENT_VERSION is being incremented COMPONENT_REVISION shall be reset.

Reply:
The `COMPONENT_VERSION` change and `COMPONENT_REVISION` addition are in two different commits though. Shall I drop it anyway?

Reply:
Pre-squashed the commits and removed `COMPONENT_VERSION`.